mobile/src/components/uam-groups-config/uam-configure-groups.tsx

```diff
@@ -0,0 +1,154 @@
+import React, { FC, useEffect } from 'react';
+import { ScrollView, View } from 'react-native';
+
+import { AppScreenName, PaginationDefaultValue } from '~/common/enums/enums';
+import { GroupsUpdateRequestDto } from '~/common/types/types';
+import { Button, Input, Text } from '~/components/common/common';
+import {
+  PermissionsTable,
+  UsersTable,
+} from '~/components/uam-groups-config/components/components';
+import {
+  useAppDispatch,
+  useAppForm,
+  useAppNavigate,
+  useAppSelector,
+  usePagination,
+  useSelectedItems,
+} from '~/hooks/hooks';
+import { groupsCreationActions, uamGroupEditActions } from '~/store/actions';
+import { groupCreateClient } from '~/validation-schemas/validation-schemas';
+
+import { CREATE_GROUP_DEFAULT_PAYLOAD } from './common/constants/create-group-default.constants';
```

Comment:
Rename create-group-default.constants to constants
```suggestion
import { CREATE_GROUP_DEFAULT_PAYLOAD } from './common/constants/constants';
```